contracts/deploy/004_single_asset_staking.js

```diff
@@ -115,9 +115,21 @@ const singleAssetStaking = async ({ getNamedAccounts, deployments }) => {
       dropProofDepth = process.env.DROP_PROOF_DEPTH;
     } else {
       // use testing generated scripts
-      const { computeRootHash } = require('../scripts/staking/airDrop.js');
-      const testPayouts = require('../scripts/staking/rawAccountsToBeCompensated.json');
-      const root = await computeRootHash(cOGNStaking.address, testPayouts);
+      const {
+        computeRootHash,
+        parseCsv,
+        compensationData
```

Comment:
`compensationData` being an export of `airDrop.js` is a bit confusing to me since it is a constant. Maybe have a `script/staking/contstants.js` file and have compensation data as an export of that.

Reply:
Done